earth2studio/models/dx/solarradiation_afno.py

```diff
@@ -0,0 +1,363 @@
+    orography : torch.Tensor
+        Surface geopotential (orography)
+    landsea_mask : torch.Tensor
+        Land sea mask
+    sincos_latlon : torch.Tensor
+        4 fields embedding location information (cos(lat), sin(lat), cos(lon), sin(lon))
+    """
+
+    def __init__(
+        self,
+        core_model: torch.nn.Module,
+        freq: str,
+        era5_mean: torch.Tensor,
+        era5_std: torch.Tensor,
+        ssrd_mean: torch.Tensor,
+        ssrd_std: torch.Tensor,
+        orography: torch.Tensor,
+        landsea_mask: torch.Tensor,
+        sincos_latlon: torch.Tensor,
+    ) -> None:
+        super().__init__()
+        self.core_model = core_model
+        self.freq = freq
+        self.register_buffer("era5_mean", era5_mean)
+        self.register_buffer("era5_std", era5_std)
+        self.register_buffer("ssrd_mean", ssrd_mean)
+        self.register_buffer("ssrd_std", ssrd_std)
+        self.register_buffer("orography", orography)
+        self.register_buffer("landsea_mask", landsea_mask)
+        self.register_buffer("sincos_latlon", sincos_latlon)
+
+    def input_coords(self) -> CoordSystem:
+        """Input coordinate system of diagnostic model
+
+        Returns
+        -------
+        CoordSystem
+            Coordinate system dictionary
+        """
+        return OrderedDict(
+            {
+                "batch": np.empty(0),
+                "time": np.empty(0),
+                "lead_time": np.empty(0),
+                "variable": np.array(VARIABLES),
+                "lat": np.linspace(90, -90, 721, endpoint=False),
+                "lon": np.linspace(0, 360, 1440, endpoint=False),
+            }
+        )
+
+    @batch_coords()
+    def output_coords(self, input_coords: CoordSystem) -> CoordSystem:
+        """Output coordinate system of diagnostic model
+
+        Parameters
+        ----------
+        input_coords : CoordSystem
+            Input coordinate system to transform into output_coords
+            by default None, will use self.input_coords.
+
+        Returns
+        -------
+        CoordSystem
+            Coordinate system dictionary
+        """
+        target_input_coords = self.input_coords()
+        handshake_dim(input_coords, "lon", 5)
+        handshake_dim(input_coords, "lat", 4)
+        handshake_dim(input_coords, "variable", 3)
+        handshake_coords(input_coords, target_input_coords, "lon")
+        handshake_coords(input_coords, target_input_coords, "lat")
+        handshake_coords(input_coords, target_input_coords, "variable")
+        output_coords = input_coords.copy()
+        output_coords["variable"] = np.array(["ssrd"])
+        return output_coords
+
+    def __str__(self) -> str:
+        return "SolarRadiationNet"
+
+    @classmethod
+    def load_default_package(cls) -> Package:
+        """Load prognostic package"""
+        package = Package(
+            "ngc://models/nvidian/onboarding/afno_dx_solarradiation@1",
```

Comment:
Update:

`ngc://models/nvidian/onboarding/afno_dx_solarradiation@1` -> `ngc://models/nvidia/earth-2/afno_dx_solarradiation@1.0.0`

Reply:
I have a PR to get the model releases now. Ill update this when its ready and test again.